app/base_functions/learning_sets.py

```diff
@@ -0,0 +1,94 @@
+from uuid import UUID
+from zoneinfo import ZoneInfo
+from typing import List, Optional
+from datetime import datetime, timedelta
+from app.tables import Card
+
+
+async def get_actual_card(
+    user_id: UUID,
+    authors: Optional[List[UUID]],
+    interval: timedelta = timedelta(seconds=300),
+):
```

Comment:
mypy: Function is missing a return type annotation

Reply:
It depends special annotation - something like "Asynchronous generator type". I don’t understand this well yet and decided not to ignore it, but leave it as a task for the future - to figure it out.

Reply:
added